app/payments.py

```diff
@@ -0,0 +1,114 @@
+from flask import Blueprint, render_template, session, jsonify, abort, url_for, redirect, flash, request
+from auth import login_required, flash_errors
+import functools
+from db import db, Teams, Players, Payments
+from passlib.hash import bcrypt_sha256
+import paypalrestsdk as paypal
+from paypalrestsdk import *
+import datetime
+import os
+
+payments_page = Blueprint('payment', __name__, url_prefix='/payment', template_folder='templates')
+
+def login_required(view):
+    @functools.wraps(view)
+    def wrapped_view(**kwargs):
+        # TODO: We should be doing check_token for this routine
+        if 'access_token' not in session:
+            flash("You must be logged in to see that!", "error")
+            return redirect(url_for('auth_page.signin'))
+
+        return view(**kwargs)
+
+    return wrapped_view
+
+paypal.configure({
+    "mode": "live",  # sandbox or live
+    "client_id": os.environ.get('client_id'),
+    "client_secret": os.environ.get('client_secret')})
+
+
+@payments_page.route('/paypal_payment', methods=['GET'])
+@login_required
+def paypal_payment():
+    # Payment
+    # A Payment Resource; create one using
+    # the above types and intent as 'sale'
+    payment = paypal.Payment({
+        "intent": "sale",
+
+        # Payer
+        # A resource representing a Payer that funds a payment
+        # Payment Method as 'paypal'
+        "payer": {
+            "payment_method": "paypal"},
+
+        # Redirect URLs
+        "redirect_urls": {
+            "return_url": "http://localhost:5000/payment/paypal_Return?success=true",
+            "cancel_url": "http://localhost:5000/payment/paypal_Return?cancel=true"},
```

Comment:
I don't think these redirect urls will work in production. The webserver won't be on localhost:5000. I'm not familiar with the API, but I believe it should at least be

http://www.collegiatecounterstrike.com/payment/paypal_Return?success=true
http://www.collegiatecounterstrike.com/payment/paypal_Return?cancel=true